clowdapp.yaml

```diff
@@ -279,6 +279,24 @@ objects:
           requests:
             cpu: 50m
             memory: 512Mi
+    # This is one time job to correct the PerformanceTable top_candidate entries
+    - name: one-time-migration-runner
+      podSpect:
+        image: ${IMAGE}:${IMAGE_TAG}
+        command: ["sh"]
+        args: ["-c", "export FLASK_APP=manage.py && poetry run flask db downgrade && poetry run flask db upgrade"]
```

Comment:
@upadhyeammit  - Could you please provide a particular revision into above commands for which you want to perform `downgrade & upgrade` ?